ngraph/test/runtime/interpreter/int_executable.hpp

```diff
@@ -1458,6 +1464,14 @@ class INTERPRETER_BACKEND_API ngraph::runtime::interpreter::INTExecutable : publ
                                        norm->get_eps_mode());
             break;
         }
+        case OP_TYPEID::NonMaxSuppression_v5:
+        {
+            const op::v5::NonMaxSuppression* nms =
+                static_cast<const op::v5::NonMaxSuppression*>(&node);
+
+            run_nms5(nms, out, args);
```

Comment:
Why do we add a special method for nms5? Looks like we don't have such methods for other operations.
Why cannot we extend reference implementation if it is needed?

Reply:
Fixed.